pom.xml

```diff
@@ -92,6 +92,24 @@
                 <artifactId>guava</artifactId>
                 <version>25.0-jre</version>
             </dependency>
+            <dependency>
+                <groupId>org.graalvm.js</groupId>
+                <artifactId>js</artifactId>
+                <version>1.0.0-rc10</version>
+                <scope>runtime</scope>
+            </dependency>
+            <dependency>
+                <groupId>org.graalvm.js</groupId>
+                <artifactId>js-scriptengine</artifactId>
+                <version>1.0.0-rc10</version>
```

Comment:
Why would we be using such old versions?

Looking at [`worker-document/pom.xml:193-L207`](https://github.com/CAFDataProcessing/worker-document/blob/v5.1.1/pom.xml#L193-L207) I see that the workers are using `20.1.0` but at this stage I'd be surprised if there isn't an even newer version than that.